### PR TITLE
GBench microbenchmarks integration

### PR DESCRIPTION
**Pull request by anisimovak**

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -72,6 +72,14 @@ mkl_repo(
     ],
 )
 
+#load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
```
Comment (amgrigoriev) [inline WORKSPACE:75]: Remove?

File: cpp/oneapi/dal/test/engine/common_gbench.hpp [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+#pragma once
+
+#include "common_base.hpp"
```
Comment (amgrigoriev) [inline cpp/oneapi/dal/test/engine/common_gbench.hpp:18]: Add new line at the end

File: cpp/oneapi/dal/test/engine/config.cpp [hunk 3]
```diff
@@ -17,7 +17,9 @@
 #include "oneapi/dal/test/engine/config.hpp"
 
 #include <stdexcept>
-#include "oneapi/dal/test/engine/common.hpp"
+#include "oneapi/dal/test/engine/common_base.hpp"
+
+#include <iostream>
```
Comment (amgrigoriev) [inline cpp/oneapi/dal/test/engine/config.cpp:22]: Is it necessary?

File: cpp/oneapi/dal/test/engine/gbench_main.cpp [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include <benchmark/benchmark.h>
+#include "oneapi/dal/test/engine/config.hpp"
+#include <iostream>
```
Comment (amgrigoriev) [inline cpp/oneapi/dal/test/engine/gbench_main.cpp:19]: Is it necessary?

File: WORKSPACE (hunk 1 above)
Comment (KulikovNikita) [reply WORKSPACE]: Fixed

**Pull request closed without merging**